cluster/juju/layers/kubernetes-worker/actions/pause

```diff
@@ -2,6 +2,18 @@
 
 set -ex
 
+FORCE_LOCAL=$(action-get force)
+
 kubectl --kubeconfig=/srv/kubernetes/config cordon $(hostname)
-kubectl --kubeconfig=/srv/kubernetes/config drain $(hostname) --force
+
+if [ ${FORCE_LOCAL} == True ]; then
```

Comment:
initial testing yielded good results, but i'm fine with changing this to be lower-case or run it through a standardize function to lowercase any trues. IIRC we're getting the GOLang values, not the jsonschema values.  Here's a snippet captured during runtime

```
# actions/restart
++ action-get force
+ FORCE=False
++ action-get docker
+ DOCKER=False
+ '[' False == True ']'
+ juju-log 'Pausing kubernetes-worker services'
+ actions/pause
++ action-get force
+ FORCE_LOCAL=False
++ hostname
+ kubectl --kubeconfig=/srv/kubernetes/config cordon juju-f3e355-1
node "juju-f3e355-1" cordoned
+ '[' False == True ']'
```